k8s/sidecar.yaml

```diff
@@ -55,10 +55,10 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 1024Mi
```

Comment:
`sidecar` doesn't really do anything memory intensive, so we are fine with 256MB.